examples/bigquery-row-access-groups/schedule_job.sh

```diff
@@ -2,10 +2,11 @@
 
 PROJECT_ID=$1
 REGION=us-central1
+SCHEDULE="11 * * * *"
```

Comment:
Why is this defaulted to schedule every 11 minutes? Is this supposed to be every 1 minute so group changes cascade to row-level access control within a minute?

Reply:
This is not running every 11 minutes, it's once every hour on the 11th minute (just an arbitrary value I chose). Since this is something a user is likely to change, I pulled it out into a variable.